apps/drec-api/src/utils/types.ts

```diff
@@ -0,0 +1,20 @@
+import { ICertificate as IOriginalCertificate } from '@energyweb/origin-247-certificate';
+
+export type EICCode = string;
+
+export enum CertificateSourceType {
+  Generator = 'generator',
+  Battery = 'battery',
+}
+
+/**
+ * Certificate source allows to describe how the certificate was issued.
+ * It can be issued by various entities like battery or generator itself
+ */
+export interface ICertificateMetadata {
+  type: CertificateSourceType;
```

Comment:
I guess we don't use that on drec, cause we don't have battery

Reply:
Yeah but I think that if I remove the type, the .issue method will fail. I can try.

Reply:
No, it won't, metadata interface is completely up to the client